docs/USAGE.md

```diff
@@ -4,7 +4,6 @@
   - [Overview](#overview)
   - [Example usage](#example-usage)
     - [More examples](#more-examples)
-    - [Gist demos](#gist-demos)
```

Comment:
I couldn't find this section

Reply:
Yeah, we wanted to list here those gists we were using on the webapp for reference purposes.